README.md

```diff
@@ -151,7 +151,28 @@ http://localhost:65432/metadata/query
 ```
 ... and expect a "200 OK" response.
 
-### Updating upstream dependencies (cardano-wallet, cardano-node, and iohk-nix)
+### Windows
+
+This option is only for troubleshooting windows specific issues with hardware wallets. It is not recommended to use Windows as a developer environment.
+Most of the commands need `nix` and will run only on Linux or macOS.
+
+#### Requisites
+- Windows 10/11
+- Daedalus testnet installation (similar version used in branch) in `C:\Program Files\Daedalus Testnet`
+- NodeJS 16
+- Python2 *
+- `yarn global add windows-build-tools` *
```

Comment:
I think you can omit this step by extracting `daedalus\nix\windows-usb-libs.zip`  under `daedalus\build` folder.